### ISSUE TITLE
Rewards for block propose alignment

### ISSUE THREAD

**Issue by davaymne** [taikoxyz/taiko-mono#14845]
### Describe the feature request

Poposer gets reward - 3 TTKOj for block proposed. 
At the same Prover set a bond - 2.5 TTKOj for block proof.

If user runs proposer and prover at the same time and gets managed propose to be the 1st in the L1 block to be aligible for TTKOj reward, then every time when it assign  block to prover, it even don't need to prove it, as it get 0.5 x TTKOj profit anyway.

While it may or may not make sense on mainnet,  it is still an opportunity for gambling Testnet.

### Describe alternatives you've considered

Description of the alternatives you've considered here.

### Additional context

Additional context here.

**Comment by dantaik** [comment]
This is a configuration issue and should be easily avoided. [This PR](https://github.com/taikoxyz/taiko-mono/pull/14924) makes it posssible to validate configs after the L2 has been initlaized.

**Comment by davidtaikocha** [comment]
Yes and we changed the A5 L1 proposing reward in each L1 block to ~1.2 TTKO around one week ago.
